src/Fhir.Anonymizer.Shared.Core/Processors/PerturbProcessor.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Fhir.Anonymizer.Core;
+using Fhir.Anonymizer.Core.Extensions;
+using Fhir.Anonymizer.Core.Models;
+using Fhir.Anonymizer.Core.Processors.Settings;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using MathNet.Numerics.Distributions;
+
+namespace Fhir.Anonymizer.Core.Processors
+{
+    public class PerturbProcessor : IAnonymizerProcessor
+    {
+        private static readonly HashSet<string> s_quantityTypeNames = new HashSet<string>
+        {
```

Comment:
Are these types shared by both stu3 and r4?

Reply:
Yeah, that's a big question. I have checked the numeric and quantity types between STU3 and R4, found that there is a slight difference that R4 has a new quantity called MoneyQuantity. So I moved this conflict definition as partial definitions to R4 and STU3 project.

Reply:
Could we simply put these parameters in Constants (it is partial already)? Not sure which one might be better

Reply:
Yes. Can't agree more. We currently can't do this because constants can only include built-in types. As we have more and more FHIR-related variables, we may need add a new class like FHIRInfo to store such informations. Thanks!